Incorrect instructions for saving settings

benhopperau opened the pull request:
Steps advised to click cancel, this would cause the instructed settings to not be saved. Have updated to click OK so settings are saved

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

Jim-Parker commented:
@coreyp-at-msft Corey, can review Aleksandar's contribution and let me know if it can be merged? Thanks

coreyp-at-msft commented:
#sign-off

Jim-Parker commented:
@alexandair Thanks for the contribution Aleksandar. I'll get these added to the article.

benhopperau committed:
Incorrect instructions for saving settings

Steps advised to click cancel, this would cause the instructed settings to not be saved. Have updated to click OK so settings are saved
